src/content/docs/alerts/overview.mdx

```diff
@@ -187,7 +187,7 @@ This diagram shows how alerts work: you create an alert policy that includes an
         </tr>        
     </tbody>
 </table>
-
+Note: During a service interruption, alerts may be unavailable.
```

Comment:
@esutherland4 Can we say more about this, or potentially change the phrasing? I would think that a service interruption is exactly when I'd want to get alerted about it

Reply:
@ally-sassman Hey! It's referencing New Relic service interruptions, so maybe below is better? This is legal requested language (related to when we suppress alerts during incidents) that we had in the docs before that was removed for some reason. Here is the old PR: https://github.com/newrelic/docs-website/pull/9707

> During a New Relic service interruption, alerts may be temporarily unavailable.